src/runtime_src/xdp/profile/plugin/user/user_cb.cpp

```diff
@@ -62,39 +58,38 @@ namespace xdp {
     if (!VPDatabase::alive() || !UserEventsPlugin::alive())
       return;
 
-    uint64_t timestamp = xrt_core::time_ns() ;
-    VPDatabase* db = userEventsPluginInstance.getDatabase() ;
+    uint64_t timestamp = xrt_core::time_ns();
+    VPDatabase* db = userEventsPluginInstance.getDatabase();
 
-    uint64_t start = (db->getDynamicInfo()).matchingStart(functionID) ;
-    VTFEvent* event = new UserRange(start, 
-                                    static_cast<double>(timestamp),
-                                    false, // isStart
-                                    0,
-                                    0) ;
+    uint64_t start = (db->getDynamicInfo()).matchingStart(functionID);
+    VTFEvent* event = new UserRange(
+        start, static_cast<double>(timestamp),
+        false, // isStart
+        0, 0);
```

Comment:
This makes sense. However some minor issues:
1. We use 2 spaces for indentation. 4 isn't consistent with that.
2. Also, I'd propose the end bracket to be on new line for even more clarity.
VTFEvent* event = new UserRange (
      start, static_cast<double>(timestamp),
      false, // isStart
      0, 0
);